plugins/provisioners/ansible/provisioner/base.rb

```diff
@@ -132,6 +132,26 @@ def extra_vars_argument
           end
         end
 
+        def get_galaxy_role_file(basedir)
+          Pathname.new(config.galaxy_role_file).expand_path(basedir)
```

Comment:
Try to be careful with Pathname; it's pretty inefficient and sometimes returns a pathname object instead of a string. _Usually_ they behave the same, but there are exceptions:

```ruby
Pathname.new("foo").expand_path
#=> #<Pathname:/Users/sethvargo/foo>
```

vs

```ruby
File.expand_path(config.galaxy_role_file, basedir)
```